### DISCUSSION TITLE
Please help with this: settings.cacheOptions.store error

### DISCUSSION THREAD

**Discussion by djsnipa1** [waylaidwanderer/node-chatgpt-api#296] [Help]
I've been fighting with this for the better part of a week and can't figure out how to get this to work correctly. I've read over all the issues and discussions in this repo. Tried a lot of different suggestions to no avail. Please, any help would be greatly appreciated! 

Thank you!

```
file:///home/chad/.npm/lib/node_modules/@waylaidwanderer/chatgpt-api/bin/cli.js:30
if (settings.storageFilePath && !settings.cacheOptions.store) {
                                                       ^

TypeError: Cannot read properties of undefined (reading 'store')
    at file:///home/chad/.npm/lib/node_modules/@waylaidwanderer/chatgpt-api/bin/cli.js:30:56

Node.js v18.15.0
```

**Comment by waylaidwanderer** [comment]
Looks like your settings.js file is missing a `cacheOptions` property?
